source/javascripts/services/_stepSourceService.js.erb

```diff
@@ -163,17 +166,17 @@ angular.module("BitriseWorkflowEditor").factory("stepSourceService", function($q
 	};
 
 	stepSourceService.loadStepsWithCVSs = function(CVSs, requestConfig) {
-		var libraryStepCVSs = _.filter(CVSs, function(aCVS) {
-			var step;
-			try {
-				step = stepSourceService.stepFromCVS(aCVS);
-			}
-			catch (error) {
-				return false;
-			}
-
-			return step.libraryURL !== undefined;
-		});
+		// var libraryStepCVSs = _.filter(CVSs, function(aCVS) {
+		// 	var step;
+		// 	try {
+		// 		step = stepSourceService.stepFromCVS(aCVS);
+		// 	}
+		// 	catch (error) {
+		// 		return false;
+		// 	}
+
+		// 	return step.libraryURL !== undefined;
+		// });
```

Comment:
can we remove this?

Reply:
I guess we can, it is not used anywhere.